lib/mock/index.js

```diff
@@ -139,6 +152,10 @@ function mockResponseHeaders(req, res, next) {
 
     if (res.swagger.headers) {
       _.forEach(res.swagger.headers, function(header, name) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (14)](https://www.codacy.com/app/jamesmessinger/swagger-express-middleware/file/2175183918/issues/source?bid=2170795&fileBranchId=3076516#l154)